doc/release/1.4.0-notes.rst

```diff
@@ -15,7 +15,7 @@ optimizations. Before upgrading, we recommend that users check that
 their own code does not use deprecated SciPy functionality (to do so,
 run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
 Our development attention will now shift to bug-fix releases on the
-1.3.x branch, and on adding new features on the master branch.
+1.4.x branch, and on adding new features on the master branch.
 
 This release requires Python 3.5+ and NumPy 1.13.3 or greater.
```

Comment:
@rgommers Are the minimum NumPy versions all still correct? We're less clear for PyPy now, right? Building from source needs 1.17.3 for Python 3.8, but that's more a development detail I think?

Reply:
Good question. py3.5+ is correct, the NumPy version isn't. From `pyproject.toml`:
```
    "numpy==1.13.3; python_version=='3.5' and platform_system!='AIX'",
    "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
    "numpy==1.14.5; python_version=='3.7' and platform_system!='AIX'",
    "numpy==1.17.3; python_version>='3.8' and platform_system!='AIX'",
```

> Building from source needs 1.17.3 for Python 3.8, but that's more a development detail I think?

No it's not a development detail, since wheels cannot be used with a version lower than we build with.

Proposed rephrase:
_NumPy >=1.13.3 (for Python 3.5, 3.6), >=1.145.5 (for Python 3.7), >= 1.17.3 (for Python 3.8)._

Reply:
Right, I figured it was a bit stale. Updated accordingly.